include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -1039,8 +1504,100 @@ namespace internal
                    is_src_locally_owned,
                    false);
     }
+
+    virtual ~PermutationFineDoFHandlerView() = default;
   };
 
+
+
+  template <int dim, int spacedim>
+  bool
+  p_transfer_without_repartitioning(
```

Comment:
I struggle with understanding the name. Can we use 
```suggestion
  p_transfer_involves_repartitioning(
```
(i.e., a positive statement) instead and use the switched notation below?